d3ledger/back-office
# D3 650 651 | Bitcoin withdrawal and deposit

# Description
Tasks is about withdrawal and deposit operations with bitcoin

#### Works done
- [x] Show bitcoin address on deposit page
- [x] Withdraw bitcoin 

# How to check
1. `yarn` to install all dependencies
2. `yarn serve` to run application
3. Check bitcoin deposit (deposit button in wallet)
4. Check bitcoin withdrawal (withdrawal button in wallet)
